ipc/ipc/sdk/src/subnet.rs

```diff
@@ -28,7 +28,7 @@ pub enum PermissionMode {
 }
 
 /// Defines the supply source of a subnet on its parent subnet.
-#[derive(Clone, Debug, Serialize_tuple, Deserialize_tuple, PartialEq, Eq)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
Why change these now?

Reply:
we are supporting lotus related now right? `Serialize_tuple` produces a different result than `Serialize`, just be consistent with other serialization methods.

Reply:
Yeah, I think they chose this in some cases to save on the field names. I think conceptually it's like `{foo: 1, bar: "hello"}` vs `[1, "hello"]`.

Reply:
I'm a bit confused because these SDK-level structs shouldn't need any kind of serialization? They are just convenience types for decoupling the contract ABIs from the application-level, right? The data from these structs is pulled into the binding-generated ABI types in `create_subnet`, but never serialized in this form?

Reply:
Bottom line: you may be able to remove the serde derive altogether here.

Reply:
It's a bit legacy code as previously it's used in lotus.

Reply:
You mean that these types in the past _were_ previously serialized to feed them into legacy or Wasm actors, right? In that case, they would've been tuple-encoded. However, I suggest you simply drop the serde derivation here all together.